Dropdown/Dropdown.js

```diff
@@ -184,37 +196,49 @@ const DropdownBase = kind({
 		}
 	},
 
-	render: ({children, css, dropdownListClassname, disabled, hasChildren, onClose, onOpen, onSelect, open, selected, transitionContainerClassname, transitionDirection, title, ...rest}) => {
+	render: ({children, className, css, dropdownButtonClassname, dropdownListClassname, disabled, hasChildren, onClose, onOpen, onSelect, open, selected, skin, transitionContainerClassname, transitionDirection, title, ...rest}) => {
 		const opened = !disabled && open;
+		const [DropDownButton, wrapperProps, groupProps] = (skin === 'silicon') ? [
+			Button,
+			{className: dropdownButtonClassname},
+			{childComponent: RadioItem, itemProps: {size: 'small', className: css.dropDownListItem, css}, selectedProp: 'selected'}
+		] : [
+			Item,
+			{},
+			{childComponent: Item, itemProps: {size: 'small'}}
+		];
 
 		return (
-			<div {...rest}>
-				<Item
-					{...rest}
-					css={css}
-					disabled={hasChildren ? disabled : true}
-					onClick={opened ? onClose : onOpen}
-				>
-					<Icon slot="slotAfter" className={css.icon} size="small">{open ? 'arrowlargeup' : 'arrowlargedown'}</Icon>
-					{title}
-				</Item>
-				<Transition
-					className={transitionContainerClassname}
-					visible={opened}
-					direction={transitionDirection}
-				>
-					<ContainerDiv className={dropdownListClassname} spotlightDisabled={!open} spotlightRestrict="self-only">
-						<Scroller className={css.scroller}>
-							<Group
-								className={css.group}
-								childComponent={Item}
-								itemProps={{size: 'small'}}
-								onSelect={onSelect}
-								selected={selected}
-							>{children || []}</Group>
-						</Scroller>
-					</ContainerDiv>
-				</Transition>
+			<div className={className} {...rest} >
+				<div {...wrapperProps}>
+					<DropDownButton
+						className={className}
```

Comment:
Good to know that you found the solution to fix the dropdown position.

But I don't think that we need to add all the same CSS classes in the `className` to the `DropDownButton`.
For now `.dropdown`, `.silicon`, `.listOpen`, and `.listClosed` are passed to the button.
1) `listClosed` will be removed.
2) `dropdown` and `silicon` might not be passed to the button.
3) So let's pass only the `.listOpen` to it after renaming it as `open` and the `className` as `buttonClassName`.

```
	computed: {
		buttonClassName: ({open, styler}) => styler.append({open}),
		...
	render: ({buttonClassName, children, css, dropdownButtonClassname, dropdownListClassname, disabled, hasChildren, onClose, onOpen, onSelect, open, selected, skin, transitionContainerClassname, transitionDirection, title, ...rest}) => {
		...

		return (
			<div {...rest} >
				<div {...wrapperProps}>
					<DropDownButton
						className={buttonClassName}
						css={css}
						disabled={hasChildren ? disabled : true}
						onClick={opened ? onClose : onOpen}
					>
						...
					</DropDownButton>
```